src/intl/messages.js

```diff
@@ -120,6 +120,8 @@ export const messages: { [messageId: string]: MessageType } = {
   },
   coresPerSockets: 'Cores per Virtual Socket',
   cpus: 'Total Virtual CPUs',
+  cpusBadTopology: 'No valid topology exists for this vCPU count.',
```

Comment:
I don't feel it is more clear, but I can change that for sure ;)

Reply:
I don't think we need to discuss that with UX folks :)

My suggestion is just to make the message suggested a bit less technical since simple users doesn't necessarily know what `vCPU`means (doesn't appear as a field on dialogs) and which topology is not valid on the basic step dialog? CPU pinning topology? 

Here is the error as appears on webadmin:
"Incorrect number of Total Virtual CPUs. It is not possible to compose this number from the available Virtual Sockets and Cores per Virtual Sockets"

Since we don't have so much space for such an informative message, we shorten it a bit but note that there are no shortcuts and message is very informative.

Reply:
Sharon's suggestions make sense.

Reply:
Done ;)